microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -3,102 +3,212 @@ description: The latest experimental features in Microsoft Edge DevTools
+> Experimental features are constantly being updated and might cause performance issues. To turn off an experimental feature, open the **Experiments** page and clear the checkbox of the experimental feature that you want to turn off.
 
 
 <!-- ordering for the h2 sections below: same order as in the Experiments page.  if change to a different ordering scheme, note that scheme here. -->
 
 
 <!-- ====================================================================== -->
-## Source order viewer
-<!-- on by default in 94 -->
+## Allow extensions to load custom stylesheets
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Capture node creation stacks
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Automatically pretty print in the Sources Panel
+
+Placeholder.
+
+
+
+<!-- ====================================================================== -->
+## Protocol Monitor
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Show CSP Violations view
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Record coverage while performance tracing
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Show option to take heap snapshot where globals are not treated as root
+
+Placeholder.
 
-**Source Order Viewer** is an experiment that displays the order of elements in the webpage source.  The on-screen display order can differ from the order of the source, which confuses screen reader and keyboard users.  Use the **Source Order Viewer** experiment to find the differences between on-screen display order and the order of the source.
 
-To use the **Source Order Viewer**:
-1.  Open the **Elements** tool.
-1.  To the right of the **Styles** tab, select the **Accessibility** tab.
-1.  Under the **Source Order Viewer** section, select the **Show Source Order** checkbox.
-1.  Highlight any HTML element to display an overlay that the order in the webpage source.
+<!-- ====================================================================== -->
+## Enable back forward cache debugging support
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Timeline: event initiators
+
+Placeholder.
+
+
+<!-- ====================================================================== -->
+## Timeline: WebGL based flamechart
+
+Placeholder.
 
-:::image type="content" source="../media/experiments-source-order-viewer.msft.png" alt-text="Source Order Viewer in the Accessibility pane" lightbox="../media/experiments-source-order-viewer.msft.png":::
 
-This experiment is available starting with Microsoft Edge version 86 and is turned on by default.
+<!-- ====================================================================== -->
+## Webassembly Debugging: Enable DWARF support
+
```

Comment:
Several headings missing here:
dual screen
APCA
full tree view
font editor

Reply:
Yes, I was on Beta, not Canary. Will fix.